docs: Audit logs (#3564)
spectrocloud/librarium: docs/docs-content/clusters/edge/local-ui/host-management/audit-logs.md

   your own application. If you changed the facility or the severity level from the suggested values, replace them with
   your actual values.

   ```conf
   $PrivDropToUser root
   $PrivDropToGroup root
   $Umask 0000
   $template ForwardFormat,"<%pri%>1 %timestamp:::date-rfc3339% %HOSTNAME% %syslogtag% %procid% - - %msg%\n"
   if ($syslogfacility-text == 'local7' and $syslogseverity-text == 'notice' and $syslogtag contains 'your-tag') then {
        action(
            type="omfile"
            file="/var/log/stylus-audit.log"
            FileCreateMode="0600"
            fileowner="root"
            template="ForwardFormat"
        )
    }
   ```

   Ensure your logs follow the RFC 5424 Syslog protocol, with the message in JSON format. The JSON object must contain
   the following keys: `edgeHostId`, `contentMsg`, `action`, `actor`, `actorType`, `resourceId`, `resourceName`,
   `resourceKind`. If your log entries do not follow this RFE 5424 Syslog protocol, or do not contain the necessary
   keys, they will still be logged, but will not show up in Local UI.

</TabItem>

<TabItem value="Send Palette Logs to Another File">

> 3. Populate the file with the following configuration. Replace `your-file` with the name of your log file.

Reviewer: @lennessyy , this should be point 4, 3 is already defined. Also point 2,3 and 4 are optional

Author: Hi Arun, so these are two tabs I made for two different use cases. One is to send stylus logs to another file, and the other is to send app logs to the stylus log file. User can do either or both. I just didn't want to create two tasks that are basically identically save for the conf file.

Can you explain what you mean by point 2, 3 ,4 are optional? That leaves step 1 which is just to clone the repo.

   if ($syslogfacility-text == 'local7' and $syslogseverity-text == 'notice' and $syslogtag contains 'stylus-audit') then {
        action(
            type="omfile"
            file="/var/log/your-file"
            FileCreateMode="0600"
            fileowner="root"
            template="ForwardFormat"
        )
    }
   ```

   This makes sure that the Palette agent logs are sent both to the file you configured and to the default
   `stylus-audit.log` file.

</TabItem>

</Tabs>

4. Follow [Build Edge Installer ISO](../../edgeforge-workflow/palette-canvos/build-installer-iso.md) to build an Edge
   installer ISO and install an Edge host. Then follow the relevant guides in
   [Deployment](../../site-deployment/site-deployment.md) to deploy a cluster with your application workloads.

### Validate

1. Log in to Local UI. For more information, refer to [Access Local UI Console](access-console.md).

2. In the left **Main Menu**, click **Audit Log**. Confirm that the logs from your applications are being collected.

## View and Download Log File from Local UI

Log files are stored in the `/var/log/` folder. The active log file `stylus-audit.log` stores up to one year's worth of

> logs. When the one period is reached, or when the file size reaches 100 MB, the log file is rotated out and archived. If

Reviewer 2: ```suggestion
logs. After a year, or when the file size reaches 100 MB, the log file is rotated out and archived. If
```

the log messages are configured to be picked up by Local UI, they will show up in Local UI. You can also download all

> log files in the `/var/log/` file from Local UI, including files that are older than one-year old, as well as files that

Reviewer 2: ```suggestion
log files in the `/var/log/` file from Local UI, including files that are older than a year, as well as files that
```

were not configured to be displayed by Local UI.

### Prerequisites

> - An active Edge host installed in the `airgap` installation mode.

Reviewer 2: ```suggestion
- An active Edge host installed in the `airgap` mode.
```

### Procedure

1. Log in to Local UI. For more information, refer to [Access Local UI Console](access-console.md).

2. In the left **Main Menu**, click **Diagnostics**. Then click the **Logs** tab. You can view all audit log entries on
   this page. Local UI will display log entries dated up to one year ago.

3. To download the log files, click the **Download** button in the upper-right corner. This will download all files in

>    the `/var/log` folder, including archived old log files as well as any additional log files you configured to be

Reviewer 2: ```suggestion
   the `/var/log` folder, including the archived log files as well as any additional log files you configured to be
```
